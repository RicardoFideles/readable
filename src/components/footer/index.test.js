import React from 'react';
import Footer from './index';

it('renders without crashing', () => {
  expect(shallow(<Footer />)).toMatchSnapshot();
});
