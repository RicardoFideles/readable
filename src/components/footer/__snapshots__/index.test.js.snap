// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<footer
  className="site-footer"
  id="colophon"
  role="contentinfo"
>
  <div
    id="footer-info"
  >
    <div
      className="container"
    >
      <div
        className="site-info"
      >
        Â© 2018 All rights reserved
        <span
          className="sep"
        >
           | 
        </span>
        Readable - Udacity - Ricardo Fideles
      </div>
    </div>
  </div>
</footer>
`;
